repos/spack_repo/builtin/packages/py_slepc4py/package.py

```diff
@@ -54,6 +54,8 @@ class PySlepc4py(PythonPackage):
 
     patch("ldshared.patch", when="@:3.18")
 
+    depends_on("c", type="build")  # generated
```

Comment:
I _think_ this was not generated by a script:
```suggestion
    depends_on("c", type="build")
```

Reply:
Can confirm.